core/process/data.loader.php

```diff
@@ -432,21 +432,33 @@
 		}
 	
 		// get all mythic pokemon
-		$req 		= "SELECT DISTINCT pokemon_id, disappear_time FROM pokemon
-				   WHERE pokemon_id IN (".implode(",", $mythic_pokemons).")
-				   ORDER BY disappear_time DESC LIMIT 0,12";
+        $req 		= "SELECT DISTINCT pokemon_id, disappear_time, latitude, longitude, individual_attack, individual_defense, individual_stamina FROM pokemon
+                    WHERE pokemon_id IN (".implode(",", $mythic_pokemons).")
+                    ORDER BY disappear_time DESC LIMIT 0,12";
 	} else {
 		// get all pokemon
-		$req		= "SELECT DISTINCT pokemon_id, disappear_time FROM pokemon ORDER BY disappear_time DESC LIMIT 0,12";
+		$req		= "SELECT DISTINCT pokemon_id, disappear_time, latitude, longitude, individual_attack, individual_defense, individual_stamina FROM pokemon ORDER BY disappear_time DESC LIMIT 0,12";
 	}
 	$result 	= $mysqli->query($req);
 	$recents	= array();
 
-	if ($result->num_rows > 0) {
-		while ($data = $result->fetch_object()) {
-			$recents[] = $data->pokemon_id;
-		}
-	}
+    if ($result->num_rows > 0) {
+        while ($data = $result->fetch_object()) {
+            $recent = new stdClass();
+            $recent->id = $data->pokemon_id;
+            $recent->last_seen = strtotime($data->disappear_time)+60*60;
```

Comment:
Yep now it's like this: https://github.com/brusselopole/Worldopole/commit/3878dd6cce7c6ece7791096c4b40abc028e7ba20
There I use the originl row to do sorting which will allow mysql to use indexes :) 
I tested it and it's even faster than the php implementation.